test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -2921,7 +2932,11 @@ TEST_F(AtenXlaTensorTest, TestOneIndexTransfer) {
     ForEachDevice([&](const torch::Device& device) {
       torch::Tensor xla_params = CopyToDevice(params, device);
       torch::Tensor xla_result = torch::index(xla_params, {indices});
-      AllClose(result, xla_result);
+      if (isFloatingType(scalar_type)) {
```

Comment:
This is a bit too chatty IMHO.
In these cases, values should really match verbatim (all indexing/select ops), so I would just use the plain expect equal values.

Reply:
Done.